src/aurora.ts

```diff
@@ -337,15 +345,14 @@ export class Aurora extends Construct {
         version,
       }),
       instanceIdentifierBase: id.pascal,
-      instanceProps: {
-        instanceType,
-        performanceInsightEncryptionKey: encryptionKey,
-        performanceInsightRetention: props.performanceInsightRetention,
-        securityGroups: this.securityGroups,
-        vpc: props.vpc,
-        vpcSubnets,
-      },
-      instances: props.instances,
+      securityGroups: this.securityGroups,
+      vpc: props.vpc,
+      vpcSubnets: vpcSubnets,
```

Comment:
Idiomatic typescript
```suggestion
      vpcSubnets,
```